waveprop/rs.py

```diff
@@ -916,7 +936,7 @@ def _bandpass(H, fX, fY, Sx, Sy, x0, y0, z0, wv):
     :param y0:
     :return:
     """
-    du = 1 / (Sx) # not divided by 2 like in 17 since field already padded
+    du = 1 / (2 * Sx)
     u_limit_p = ((x0 + 1 / (2 * du)) ** (-2) * z0**2 + 1) ** (-1 / 2) / wv
     u_limit_n = ((x0 - 1 / (2 * du)) ** (-2) * z0**2 + 1) ** (-1 / 2) / wv
     if Sx < x0:
```

Comment:
but actually this should rather be ``Sx / 2 < x0`` since Sx is padded (and same with Sy/2) below.